airflow/gcp/hooks/pubsub.py

```diff
@@ -52,18 +57,39 @@ class PubSubHook(GoogleCloudBaseHook):
 
     def __init__(self, gcp_conn_id: str = 'google_cloud_default', delegate_to: str = None) -> None:
         super().__init__(gcp_conn_id, delegate_to=delegate_to)
+        self._client = None
 
-    def get_conn(self) -> Any:
+    def get_conn(self) -> PublisherClient:
         """
-        Returns a Pub/Sub service object.
+        Retrieves connection to Google Cloud Pub/Sub.
 
-        :rtype: googleapiclient.discovery.Resource
+        :return: Google Cloud Pub/Sub client object.
+        :rtype: google.cloud.pubsub_v1.PublisherClient
         """
-        http_authorized = self._authorize()
-        return build(
-            'pubsub', 'v1', http=http_authorized, cache_discovery=False)
+        if not self._client:
+            self._client = PublisherClient(
+                credentials=self._get_credentials(),
+            )
+        return self._client
 
-    def publish(self, project: str, topic: str, messages: List[Dict]) -> None:
+    @cached_property
+    def subscriber_client(self) -> SubscriberClient:
+        """
+        Creates SubscriberClient.
+
+        :return: Google Cloud Pub/Sub client object.
+        :rtype: google.cloud.pubsub_v1.SubscriberClient
+        """
+        return SubscriberClient(
+            credentials=self._get_credentials(),
```

Comment:
Can you add client_info here?

Reply:
`SubscriberClient` doesn't get `client_info` argument.

Reply:
I was wrong. As you showed me, it takes later from kwargs.